### DISCUSSION TITLE
Example for using Non-blocking delayed retries with Kafka streams

### DISCUSSION THREAD

**Discussion by austiine04** [spring-projects/spring-kafka#1878] [Q&A] [answered]
Hi there, I would like to use the new non-blocking delayed retries feature within a kafka streams application, is there an example of this somewhere?

Thank you

**Comment by garyrussell** [comment, marked as answer]
There is no support for Kafka Streams - Spring for Apache Kafka support for Kafka Streams is limited to the factory bean for setting up the stream the Spring way, as well as providing a couple of transformers.

Spring is not involved at runtime at all.

**Reply by austiine04** [reply]
Thanks for your answer, and am sorry for posting this question to stack overflow as well, did not realise you already answered won't do that again.
